### PR TITLE
Updating point releases for SES 6

### LINKED ISSUE #710: SES6 Admin Guide: "Ceph Maintenance Updates Based on Upstream 'Nautilus' Point Releases" appendix is missing 3 upstream releases
**Link To Section**
http://docserv.nue.suse.com/documents/SES_6/ses-admin/single-html/#id-1.3.10

**Summary**
The "Upstream 'Nautilus' Point Releases" is missing sections for upstream releases:

* 14.2.10 - https://ceph.io/releases/v14-2-10-nautilus-released/#notable-changes
* 14.2.11 - https://ceph.io/releases/v14-2-11-nautilus-released/#notable-changes
* 14.2.12 - https://ceph--37718.org.readthedocs.build/en/37718/releases/nautilus/#notable-changes

Please add them. I provided links to the "Notable changes" section of the relevant upstream release notes.

### PR DESCRIPTION
**Pull request by asettle**
Closes #710

### CODE REVIEW

File: xml/ceph_maintenance_updates.xml [hunk 1]
```diff
@@ -33,6 +33,145 @@
...
+       added. By default, it will set the repair count to 0. If you want to
+       be warned again if additional repairs are performed, you can provide a
+       value to the command and specify the value of
+       <literal>mon_osd_warn_num_repaired</literal>. This command will be replaced
+       in future releases by the health mute/unmute feature.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       It is now possible to specify the initial monitor to contact for &ceph;
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:63]:
```suggestion
       It is now possible to specify the initial MON to contact for &ceph;
```

File: xml/ceph_maintenance_updates.xml [hunk 2]
```diff
@@ -33,6 +33,145 @@
...
+       tools and daemons using the <option>mon_host_override config</option> option
+       or <option>--mon-host-override <replaceable>ip</replaceable></option>
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:65]:
```suggestion
       or <option>--mon-host-override <replaceable>IP</replaceable></option>
```

File: xml/ceph_maintenance_updates.xml [hunk 3]
```diff
@@ -33,6 +33,145 @@
...
+       command-line switch. This generally should only be used for debugging
+       and only affects initial communication with &ceph;’s monitor cluster.
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:67]:
```suggestion
       and only affects initial communication with &ceph;’s MON cluster.
```

File: xml/ceph_maintenance_updates.xml [hunk 4]
```diff
@@ -33,6 +33,145 @@
...
+     <para>
+       RGW: Bucket notifications now support Kafka endpoints. This requires
+       librdkafka of version 0.9.2 and up. Note that Ubuntu 16.04.6 LTS
+       (Xenial Xerus) has an older version of librdkafka, and would require
+       an update to the library.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       The pool parameter <literal>target_size_ratio</literal>, used by the pg
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:133]:
```suggestion
       The pool parameter <option>target_size_ratio</option>, used by the PG
```

File: xml/ceph_maintenance_updates.xml [hunk 5]
```diff
@@ -33,6 +33,145 @@
...
+       autoscaler, has changed meaning. It is now normalized across pools,
+       rather than specifying an absolute ratio. If you have set target size
+       ratios on any pools, you may want to set these pools to autoscale <literal>warn</literal>
+       mode to avoid data movement during the upgrade:
+     </para>
+<screen>ceph osd pool set <replaceable>pool-name</replaceable> pg_autoscale_mode warn</screen>
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:139]:
```suggestion
<screen>ceph osd pool set <replaceable>POOL_NAME</replaceable> pg_autoscale_mode warn</screen>
```

File: xml/ceph_maintenance_updates.xml [hunk 6]
```diff
@@ -33,6 +33,145 @@
...
+ </para>
+ <itemizedlist>
+   <listitem>
+     <para>
+       The <command>ceph df command</command> now lists the number of pgs in each pool.
+     </para>
+   </listitem>
+   <listitem>
+     <para>
+       Monitors now have a config option <option>mon_osd_warn_num_repaired</option>,
```
Comment (tbazant) [inline xml/ceph_maintenance_updates.xml:49]:
```suggestion
       MONs now have a config option <option>mon_osd_warn_num_repaired</option>,
```

**Review by tbazant — CHANGES_REQUESTED** [review]
several nits, accept or deny :-)

**Review by tbazant — APPROVED** [review]
lgtm

File: xml/ceph_maintenance_updates.xml [hunk 7]
```diff
@@ -33,6 +33,145 @@
...
+   In addition to security flaws, this major upstream release brought a number of
+   notable changes:
+ </para>
+ <itemizedlist>
+   <listitem>
+     <para>
+       RGW: Bucket notifications now support Kafka endpoints. This requires
+       librdkafka of version 0.9.2 and up. Note that Ubuntu 16.04.6 LTS
+       (Xenial Xerus) has an older version of librdkafka, and would require
+       an update to the library.
```
Comment (smithfarm) [inline xml/ceph_maintenance_updates.xml:128]: This "Kafka endpoints" note is not applicable to our product, so it should be dropped.

**Pull request merged**